pkg/controller/reconciler/mock_radius_client_test.go

```diff
@@ -75,6 +76,13 @@ func (rc *mockRadiusClient) Update(exec func()) {
 	exec()
 }
```

Comment:
The changes in this file are for clarity. I was getting very confused when adding a new client, so I made it explicit that these operations are for the RadiusClient